### DISCUSSION TITLE
Image syntax

### DISCUSSION THREAD

**Discussion by liliaoyuan** [microsoft/vscode-jupyter#15548] [Questions and Answers]
Hi there 
I want to insert a picture into a markdown cell
As I search as much as I can, I know there are two methods
markdorwn way:`![](path)`
or 
html: \<img src="path">
![IG)W@H}FK 5SQKB 3G5ZH1I](https://github.com/microsoft/vscode-jupyter/assets/34170909/7bac096e-add5-4725-abe5-242e659a4f97)
![741GSUP%D} VND3~NGANS0I](https://github.com/microsoft/vscode-jupyter/assets/34170909/f680b3b0-b8d2-49e5-b237-e05db910b09f)

However the things is, for both sides, I can not use the absolute path of my figure( A Local )
see the figure above.

The first screenshot is showing the path, and the first <img src = "current path"> works fine, 
while the second one with absolute path finding nothing

**Comment by DonJayamanne** [comment]
@amunger /cc

**Comment by amunger** [comment]
That's a markdown limitation in general, vscode doesn't allow images from outside the workspace to be rendered since they can come from untrusted locations. There isn't any extra logic to check if a full filepath is within the current workspace, so those also won't render.

**Reply by liliaoyuan** [reply]
thanks,
BTW, a primary question relatted: Regarding their URL handling scheme, I believe they support it, but I'm curious about how they assess the reliability of these external links
